custom/icds_reports/management/commands/run_aggregation_query.py

```diff
@@ -98,7 +98,7 @@ def run_task(agg_record, query_name):
     query = function_map[query_name]
     if query.by_state == SINGLE_STATE:
         greenlets = []
-        pool = Pool(10)
+        pool = Pool(15)
```

Comment:
What if we created a django model for aggregation settings? Could be just a basic key value pair, but it would make changing these sort of things on the fly much easier

Reply:
to be more clear I'm thinking that this could look something like:

```python
    pool = Pool(IcdsAggSettings.objects.get(key=STATE_QUERY_SLUG).value)
```

and the interface to change that could just be the django admin page